lib/reversetunnel/agent.go

```diff
@@ -354,16 +354,25 @@ func (a *Agent) handleGlobalRequests(ctx context.Context, requestCh <-chan *ssh.
 
 			switch r.Type {
 			case versionRequest:
-				err := r.Reply(true, []byte(teleport.Version))
+				// reply with the auth server version
+				pong, err := a.Client.Ping(ctx)
 				if err != nil {
-					log.Debugf("Failed to reply to %v request: %v.", r.Type, err)
+					a.log.WithError(err).Debugf("Failed to ping auth server.")
```

Comment:
Can we increase severity level to INFO ? 
In case where the ping call failed it hole addRemoteCluster operation fails so it would be nice to have insight about this.